src/test/bye/bye.py

```diff
@@ -1 +1 @@
-
+print(1)
```

Comment:
No magic numbers please.
```suggestion
try: 0/0
except Exception as e: print(int(bool(e)))
```
(This one only has emoticons.)